semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolOps.scala

```diff
@@ -1,16 +1,20 @@
 package scala.meta.internal.scalacp
 
 import java.util.{HashMap, HashSet}
+import scala.meta.internal.metacp._
+import scala.collection.mutable
 import scala.meta.internal.{semanticdb => s}
 import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
-import scala.meta.internal.semanticdb.Scala.{Names => n}
 import scala.meta.internal.semanticdb.SymbolInformation.{Kind => k}
 import scala.tools.scalap.scalax.rules.scalasig._
 
-trait SymbolOps { self: Scalacp =>
+trait SymbolOps { scalacp: Scalacp =>
   lazy val symbolCache = new HashMap[Symbol, String]
+  lazy val isJavaDefined = mutable.Map.empty[Symbol, Boolean]
```

Comment:
`private lazy val javaDefinedCache`? This way, you won't have to disambiguate with the `javaDefined` method.

Reply:
Done.